packages/manager/apps/support/index.js

```diff
@@ -0,0 +1,20 @@
+import ovhUxManagerSupport from '@ovh-ux/manager-support';
+import ovhUxNgOvhOtrs from '@ovh-ux/ng-ovh-otrs';
+import uiRouterAngularJs from '@uirouter/angularjs';
+import angular from 'angular';
+
+/* eslint-disable import/no-webpack-loader-syntax, import/extensions */
+import 'script-loader!jquery';
+/* eslint-enable import/no-webpack-loader-syntax, import/extensions */
+
+import { registerState } from './index.routing';
+
+angular
+  .module('supportApp', [
+    ovhUxManagerSupport,
+    ovhUxNgOvhOtrs,
+    uiRouterAngularJs,
```

Comment:
Sounds a bit obvious as we're in an `angular.module` declaration, don't you think ?

Reply:
It's the name of the import, didn't wan't to have to think about the naming of my variable